src/file_io.py

```diff
@@ -1,12 +1,21 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
+import io
+import json
+import logging
 import os
 import pickle
 import re
 from gettext import gettext as _
+from pathlib import Path
 from xml.dom import minidom
 
+from gi.repository import GLib
+
 from graphs import utilities
 
+from matplotlib import RcParams, cbook
+from matplotlib.style.core import STYLE_BLACKLIST
```

Comment:
So according to the docs, STYLE_BLACKLIST contains the rcparams that are not updated.
Is this list empty by default? What does it usually contain?

Reply:
STYLE_BLACKLIST is a set of parameters like backend, etc. which must not be changed by a style. These have nothing to do with visual appearance. They can be added to rcparams just fine however which is why the upstream file parser has this ignorelist which I'm just referencing here